Devscord.DiscordFramework/Integration/ServerInitializer.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Devscord.DiscordFramework.Integration;
+using Devscord.DiscordFramework.Integration.Services;
+using Discord.WebSocket;
+using Serilog;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Devscord.DiscordFramework.Framework
+{
+    public static class ServerInitializer
+    {
+        public static bool Initialized { get; private set; }
+
+        public static void Initialize(DiscordSocketClient client)
+        {
+            if (Initialized)
+            {
+                return;
+            }
+            var sw = new Stopwatch();
```

Comment:
jeśli dobrze pamiętam to da się zrobić coś w stylu var sw = Stopwatch.StartNew()